### PR TITLE
feat: added E2E tests for scheduled crypto approve allowance transact…

### PR DESCRIPTION
**Pull request by tdermendzhievv**
**Description**:
This PR introduces new E2E tests which are testing scheduled crypto approve allowance transactions.

**Related issue(s)**: #3762 

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/CryptoApproveAllowanceSuite.java [hunk 1]
```diff
@@ -1756,6 +1779,209 @@ private HapiApiSpec cannotHaveMultipleAllowedSpendersForTheSameNFTSerial() {
...
+                                                        false,
+                                                        List.of(1L, 2L, 3L))
+                                                .fee(ONE_HUNDRED_HBARS))
+                                .via("successTx"))
+                .then(
+                        cryptoTransfer(
+                                        movingUniqueWithAllowance(NON_FUNGIBLE_TOKEN, 3)
+                                                .between(OWNER, OTHER_RECEIVER))
+                                .payingWith(SPENDER)
+                                .signedBy(SPENDER)
```
Comment (IvanKavaldzhiev) [inline test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/CryptoApproveAllowanceSuite.java:1852]: signedBy is obsolete

File: test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/CryptoApproveAllowanceSuite.java [hunk 2]
```diff
@@ -1756,6 +1779,209 @@ private HapiApiSpec cannotHaveMultipleAllowedSpendersForTheSameNFTSerial() {
...
+                                .hasKnownStatus(SPENDER_DOES_NOT_HAVE_ALLOWANCE),
+                        cryptoTransfer(
+                                        movingWithAllowance(50, FUNGIBLE_TOKEN)
+                                                .between(OWNER, RECEIVER))
+                                .payingWith(SPENDER)
+                                .signedBy(SPENDER)
```
Comment (IvanKavaldzhiev) [inline test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/CryptoApproveAllowanceSuite.java:1858]: signedBy is obsolete

File: test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/CryptoApproveAllowanceSuite.java [hunk 3]
```diff
@@ -1756,6 +1779,209 @@ private HapiApiSpec cannotHaveMultipleAllowedSpendersForTheSameNFTSerial() {
...
+                                .hasKnownStatus(SPENDER_DOES_NOT_HAVE_ALLOWANCE),
+                        cryptoTransfer(allowanceTinyBarsFromTo(OWNER, RECEIVER, 5 * ONE_HBAR))
+                                .payingWith(SPENDER)
+                                .signedBy(SPENDER)
```
Comment (IvanKavaldzhiev) [inline test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/CryptoApproveAllowanceSuite.java:1862]: signedBy is obsolete

File: test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/CryptoApproveAllowanceSuite.java [hunk 4]
```diff
@@ -1756,6 +1779,209 @@ private HapiApiSpec cannotHaveMultipleAllowedSpendersForTheSameNFTSerial() {
...
+                                .hasKnownStatus(SPENDER_DOES_NOT_HAVE_ALLOWANCE),
+                        scheduleSign(SCHEDULED_TXN).alsoSigningWith(OWNER),
+                        cryptoTransfer(
+                                        movingUniqueWithAllowance(NON_FUNGIBLE_TOKEN, 3)
+                                                .between(OWNER, OTHER_RECEIVER))
+                                .payingWith(SPENDER)
+                                .signedBy(SPENDER),
```
Comment (IvanKavaldzhiev) [inline test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/CryptoApproveAllowanceSuite.java:1869]: signedBy is obsolete

File: test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/CryptoApproveAllowanceSuite.java [hunk 5]
```diff
@@ -1756,6 +1779,209 @@ private HapiApiSpec cannotHaveMultipleAllowedSpendersForTheSameNFTSerial() {
...
+                        getAccountBalance(OTHER_RECEIVER).hasTokenBalance(NON_FUNGIBLE_TOKEN, 1),
+                        cryptoTransfer(
+                                        movingWithAllowance(50, FUNGIBLE_TOKEN)
+                                                .between(OWNER, RECEIVER))
+                                .payingWith(SPENDER)
+                                .signedBy(SPENDER),
```
Comment (IvanKavaldzhiev) [inline test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/CryptoApproveAllowanceSuite.java:1875]: signedBy is obsolete

**Review by tinker-michaelj — DISMISSED** [review]
LGTM

**Review by stoqnkpL — DISMISSED** [review]
LGTM

**Review by tinker-michaelj — APPROVED** [review]
LGTM!

**Pull request merged**